docs/modules/ROOT/pages/interfaces.adoc

```diff
@@ -0,0 +1,177 @@
+
+```javascript
+#[starknet::interface]
+trait IERC20<TState> {
+    fn name(self: @TState) -> felt252;
+    fn symbol(self: @TState) -> felt252;
+    fn decimals(self: @TState) -> u8;
+    fn total_supply(self: @TState) -> u256;
+    fn balance_of(self: @TState, account: ContractAddress) -> u256;
+    fn allowance(self: @TState, owner: ContractAddress, spender: ContractAddress) -> u256;
+    fn transfer(ref self: TState, recipient: ContractAddress, amount: u256) -> bool;
+    fn transfer_from(
+        ref self: TState, sender: ContractAddress, recipient: ContractAddress, amount: u256
+    ) -> bool;
+    fn approve(ref self: TState, spender: ContractAddress, amount: u256) -> bool;
+}
+```
+
+=== ABI traits (`ERC20ABI`)
+
+They describe a contract's interface. This is useful to interface with a preset contract offered by this library, such as the ERC20 preset that includes non-standard functions like `increase_allowance`.
+
+```javascript
+#[starknet::interface]
+trait ERC20ABI<TState> {
+    fn name(self: @TState) -> felt252;
+    fn symbol(self: @TState) -> felt252;
+    fn decimals(self: @TState) -> u8;
+    fn total_supply(self: @TState) -> u256;
+    fn balance_of(self: @TState, account: ContractAddress) -> u256;
+    fn allowance(self: @TState, owner: ContractAddress, spender: ContractAddress) -> u256;
+    fn transfer(ref self: TState, recipient: ContractAddress, amount: u256) -> bool;
+    fn transfer_from(
+        ref self: TState, sender: ContractAddress, recipient: ContractAddress, amount: u256
+    ) -> bool;
+    fn approve(ref self: TState, spender: ContractAddress, amount: u256) -> bool;
+    fn increase_allowance(ref self: TState, spender: ContractAddress, added_value: u256) -> bool;
+    fn decrease_allowance(
+        ref self: TState, spender: ContractAddress, subtracted_value: u256
+    ) -> bool;
+}
+```
+
+=== Dispatcher traits (`DualERC20`)
+This is a utility trait to interface with contracts whose interface is unknown. Read more in the xref:#dualcase_dispatchers[DualCase Dispatchers] section.
+
+```javascript
+#[derive(Copy, Drop)]
+struct DualERC20 {
+    contract_address: ContractAddress
+}
+
+trait DualERC20Trait {
+    fn name(self: @DualERC20) -> felt252;
+    fn symbol(self: @DualERC20) -> felt252;
+    fn decimals(self: @DualERC20) -> u8;
+    fn total_supply(self: @DualERC20) -> u256;
+    fn balance_of(self: @DualERC20, account: ContractAddress) -> u256;
+    fn allowance(self: @DualERC20, owner: ContractAddress, spender: ContractAddress) -> u256;
+    fn transfer(self: @DualERC20, recipient: ContractAddress, amount: u256) -> bool;
+    fn transfer_from(
+        self: @DualERC20, sender: ContractAddress, recipient: ContractAddress, amount: u256
+    ) -> bool;
+    fn approve(self: @DualERC20, spender: ContractAddress, amount: u256) -> bool;
+}
+```
+
+== Dual Interfaces
```

Comment:
```suggestion
== Dual interfaces
```
Even though I agree with the capital `Interfaces`, it's not consistent with the current doc style https://github.com/OpenZeppelin/cairo-contracts/issues/726#issuecomment-1718384827